### ISSUE TITLE
[IntelliJ][ReportedByUser] Azure Plugin for IntelliJ Error

### ISSUE THREAD

**Issue by askvortsov** [microsoft/azure-tools-for-java#3520]
IntelliJ build version: 2018.3.4 IC-183.5429.30
OS: Linux
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.23.0-2018.3
Error message: 
```
rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: com.microsoft.azure.hdinsight.sdk.common.livy.interactive.SparkSession.class
	at rx.exceptions.OnErrorThrowable.addValueAsLastCause(OnErrorThrowable.java:118)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:73)
	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
	at rx.internal.operators.OperatorDoAfterTerminate$1.onNext(OperatorDoAfterTerminate.java:50)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:77)
	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:77)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.emitScalar(OperatorMerge.java:395)
	at rx.internal.operators.OperatorMerge$MergeSubscriber.tryEmit(OperatorMerge.java:355)
	at rx.internal.operators.OperatorMerge$InnerSubscriber.onNext(OperatorMerge.java:846)
	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
	at rx.internal.operators.OperatorDoAfterTerminate$1.onNext(OperatorDoAfterTerminate.java:50)
	at rx.internal.util.ScalarSynchronousObservable$WeakSingleProducer.request(ScalarSynchronousObservable.java:276)
	at rx.Subscriber.setProducer(Subscriber.java:211)
	at rx.Subscr

**Comment by vscjavaci** [comment]
(Experimental duplicate detection)
Thanks for submitting this issue. Please check if it is already covered by an existing one:
- [Report to Microsoft][SQL Server big data]Get error when submit and remotely run with 52.179.143.122... (#3534)

**Comment by t-rufang** [comment]
Duplicate with #3534 and fixed in #3638
